esp-wifi/src/wifi/mod.rs

```diff
@@ -1770,10 +1780,208 @@ fn convert_ap_info(record: &include::wifi_ap_record_t) -> AccessPointInfo {
     }
 }
 
+#[cfg(not(any(esp32c6)))]
```

Comment:
I know it's not related to this PR, but @bjoernQ do you know why this is unavailable on the C6?